<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainSettings</class>
 <widget class="QDialog" name="MainSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="chkSwipeUpYPlus">
           <property name="text">
            <string>Swipe Up (Y+) </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkSwipeDownYMinus">
           <property name="text">
            <string>Swipe Down (Y-) </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkSwipeRightXPlus">
           <property name="text">
            <string>Swipe Right (X+)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkSwipeLeftXMinus">
           <property name="text">
            <string>Swipe Left (X-)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QKeySequenceEdit" name="kseSwipeUpYPlus">
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
           <property name="maximumSequenceLength">
            <longLong>1</longLong>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QKeySequenceEdit" name="kseSwipeDownYMinus">
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
           <property name="maximumSequenceLength">
            <longLong>1</longLong>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QKeySequenceEdit" name="kseSwipeRightXPlus">
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
           <property name="maximumSequenceLength">
            <longLong>1</longLong>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QKeySequenceEdit" name="kseSwipeLeftXMinus">
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
           <property name="maximumSequenceLength">
            <longLong>1</longLong>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnSave">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnClose">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
